packages/twenty-front/src/modules/command-menu/pages/page-layout/components/ChartSettings.tsx

```diff
@@ -148,6 +150,56 @@ export const ChartSettings = ({ widget }: { widget: PageLayoutWidget }) => {
               });
             };
 
+            const createRangeValidator = (itemId: string) => {
```

Comment:
change: this is too complex, why do we need a function that returns a function, just call the function with itemId.